js/.jscsrc

```diff
@@ -5,6 +5,7 @@
   "disallowMultipleLineStrings": true,
   "disallowMultipleVarDecl": true,
   "disallowQuotedKeysInObjects": "allButReserved",
+  //"disallowSemicolons": true,
```

Comment:
I tried using that option too when it came out but it also won't work because of the semicolons we have at the end of each file.